upload/admin/view/javascript/smailyforopencart/smaily_admin.js

```diff
@@ -177,5 +177,30 @@
         },
       });
     });
+    $(".smaily-rss-options").change(function (event) {
+      var rss_url_base = smaily_rss_url_base + '&';
```

Comment:
Is it guaranteed that the base URL will always be `http://example.com/index.php?route=extension/smailyforopencart/rss`, i.e. it will contain a question mark?

Reply:
I looked into this by enabling SEO-Friendly URLs, but I realized that it's mainly meant for products and it won't create a base URL like http://example.com/smailyforopencart/rss. If the shop owner has manually created a custom URL for the smaily module, then it might cause problems.

Reply:
OK. Then this is off the table.